packages/contentful--create-contentful-app/README.md

```diff
@@ -1,19 +1,87 @@
+<!-- Don't forget to also update the corresponding README.md of the create-contentful-app package -->
+
 # Create Contentful App
 
-This project makes it easy to bootstrap [Contentful Apps](https://www.contentful.com/developers/docs/extensibility/app-framework/) with [React](https://reactjs.org/) and [Forma 36](https://f36.contentful.com/) (Contentful's design system).
+`create-contentful-app` is a command line interface to easily bootstrap [Contentful Apps](https://www.contentful.com/developers/docs/extensibility/app-framework/).
 
-## Requirements
+# Requirements
 
 - Node.js v14.15 or v16
-- NPM v6
+- npm v6 or later
+
+# Usage
+
+To start developing your first app, run:
+
+```bash
+npx create-contentful-app my-first-app
+```
+
+![Screenshot of `npx create-contentful-app my-app`](./docs/screenshot.png)
+
+## Bootstrap
 
+You can run `create-contentful-app` using one of the bollowing commands:
 
-# Quick Overview
+```bash
+# npx
+npx create-contentful-app <app-name>
 
-To start developing your app:
+# npm
+npm init contentful-app <app-name>
 
+# Yarn
+yarn create contentful-app <app-name>
 ```
-npx @contentful/create-contentful-app my-first-app
-cd my-first-app
-npm start
+
+## Package Manager
+
+`--npm` & `--yarn`
+
+Use npm or Yarn to manage dependencies. If omitted, defaults to the manager used to run `create-contentful-app`.
```

Comment:
Any reason why yarn is capitalized and npm not?

Reply:
That's the official capitalization